README.md

```diff
@@ -34,6 +34,8 @@ module.exports = {
 }
 ```
 
+Also there is a optional option eventsOptions, read the section 'eventsOptions' to know how to used it.
```

Comment:
I think this sentence can be safely removed, as the relevant section is not that far away below.

Reply:
Ok, I will remove it.

Reply:
```suggestion
```

Reply:
Removed.